qa/tasks/rgw.py

```diff
@@ -1121,53 +1114,26 @@ def task(ctx, config):
     overrides = ctx.config.get('overrides', {})
     teuthology.deep_merge(config, overrides.get('rgw', {}))
 
-    regions = {}
-    if 'regions' in config:
-        # separate region info so only clients are keys in config
-        regions = config['regions']
-        del config['regions']
+    regions = config.pop('regions', {})
```

Comment:
yeah, i have a https://github.com/cbodley/ceph/commits/wip-rgw-agent-cleanup branch that removes this stuff